spec_parser/utils.py

```diff
@@ -268,26 +268,28 @@ def _extract_entries(self, entry_list):
             self.entries[_key] = _value
 
     def _gen_uri(self, entity):
-
-        splitted = re.split(r":", entity)
-
-        if len(splitted) > 2:
+        if getattr(self, "spec", None) is None:
             return Literal(entity)
 
-        if getattr(self, "spec", None) is None:
+        if ":" in entity:
+            namespace_and_entity = re.split(r":", entity)
+        elif "/" in entity:
+            namespace_and_entity = re.split(r"/", entity)[1:]
```

Comment:
this drops the first empty entry? Maybe instead filter for `!= ""` or only drop if is empty.

Reply:
Done